docs/source/gbd2017_models/risks/low_birthweight_short_gestation/index.rst

```diff
@@ -51,3 +52,162 @@ described below, and relative to the joint TMREL.
+
+.. note::
+   the TMREL categories listed in GBD 2017 risk appendix are wrong.  
+
+Causes that are affected by LBWSG
+---------------------------------
+
+For GBD 2017, the relative risk of all-cause mortality across all available sources was analysed and outcomes based on criteria of biologic plausibility were selected. Some causes, most
+notably congenital birth defects, haemoglobinopathies, malaria, and HIV/AIDS, were excluded based on the criteria that reverse causality could not be excluded
+
++----------+---------------------------------------------------------+
+| Cause id | Cause                                                   | 
++==========+=========================================================+
+|  302     | diarrheal diseases                                      |
++----------+---------------------------------------------------------+
+|  322     | lower respiratory tract infections                      |
++----------+---------------------------------------------------------+
+|  328     | upper respiratory tract infections                      | 
++----------+---------------------------------------------------------+
+|  329     | otitis media                                            |
++----------+---------------------------------------------------------+
+|  333     | pneumococcal meningitis                                 |
++----------+---------------------------------------------------------+
+|  334     | H influenzae type B meningitis                          |
++----------+---------------------------------------------------------+
+|  335     | meningococcal meningitis                                |
++----------+---------------------------------------------------------+
+|  336     | other meningitis                                        |
++----------+---------------------------------------------------------+
+|  337     | encephalitis                                            |
++----------+---------------------------------------------------------+
+|  381     | neonatal preterm birth complications                    |
++----------+---------------------------------------------------------+
+|  382     | neonatal encephalopathy due to birth asphyxia and trauma|
++----------+---------------------------------------------------------+
+|  383     | neonatal sepsis and other neonatal infections           |
++----------+---------------------------------------------------------+
+|  384     | hemolytic disease and other neonatal jaundice           |
++----------+---------------------------------------------------------+
+|  385     | other neonatal disorders                                |
++----------+---------------------------------------------------------+
+|  686     | sudden infant death syndrome                            |
++----------+---------------------------------------------------------+
```

Comment:
Again, I think a list-table would be easier to format here, but perhaps it's not worth changing at this point.